autogen_omero.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -e
+set -u
+set -x
+
+# TODO
+echo no linkcheck
+
+# from the sub-script
+export WORKSPACE=${WORKSPACE:-$(pwd)}
+export USER=${USER:-$(whoami)}
+export OMERODIR=${WORKSPACE}/OMERO.server
+
+# VARIABLES #1
+MESSAGE="Update auto-generated documentation"
+PUSH_COMMAND="update-submodules develop --no-ask --push develop/latest/autogen"
+OPEN_PR=false
+export SPHINXOPTS=-W
+
+# Responsibilities of caller, likely omero-docs-superbuild
+test -e $WORKSPACE/OMERO.server
+test -e $WORKSPACE/omero-install
+test -e $WORKSPACE/omeroweb-install
+
+if [ -e $WORKSPACE/venv ]; then
+    rm -rf $WORKSPACE/venv
+fi
+python3 -m venv --system-site-packages $WORKSPACE/venv || virtualenv --system-site-packages $WORKSPACE/venv
+$WORKSPACE/venv/bin/pip install -r $WORKSPACE/OMERO.server/share/web/requirements-py27.txt
```

Comment:
``$WORKSPACE/venv/bin/pip install -r $WORKSPACE/OMERO.server/share/web/requirements-py27.txt`` seems strange with the decoupled work

Reply:
Agreed. Happy to replace it, but requires something upstream.
see https://github.com/ome/omero-web/issues/96